tasks/create.yaml

```diff
@@ -50,7 +58,12 @@ tasks:
     description: "Create the K3d + UDS Core Checkpoint Zarf Package"
     actions:
       - description: "Create the UDS Core Checkpoint Zarf Package"
-        cmd: "uds zarf package create packages/checkpoint-dev --confirm --no-progress --skip-sbom"
+        task: common:package
+        with:
+          path: packages/checkpoint-dev
+          config: ./zarf-config.yaml
+          options: "${CREATE_OPTIONS} --skip-sbom"
```

Comment:
I think this makes sense for checkpoint - @Racer159 concur? I don't think there's any need/usecase for the sbom on the checkpoint package (and I'd wonder if it even provides any value given the composition of that package).